rest/networkProtocols/TheThingsNetwork.js

```diff
@@ -17,25 +17,37 @@ module.exports = {
       oauthUrl: 'https://account.thethingsnetwork.org/users/authorize',
       protocolHandlerNetworkFields: [
         {
-          fieldName: 'clientId',
-          fieldDesc: 'Client Id',
-          fieldHelp: 'The client id chosen when registering LPWan',
-          fieldType: 'string',
-          displayWithQueryParameter: ''
+          name: 'clientId',
+          description: 'Client Id',
+          help: 'The client id chosen when registering LPWan',
+          type: 'string',
+          label: 'Client ID',
+          value: '',
+          required: true,
+          placeholder: 'TTN Client ID',
+          oauthQueryParameter: ''
         },
         {
-          fieldName: 'clientSecret',
-          fieldDesc: 'Client Secret',
-          fieldHelp: 'The client secret provided when registering LPWan',
-          fieldType: 'string',
-          displayWithQueryParameter: ''
+          name: 'clientSecret',
+          description: 'Client Secret',
+          help: 'The client secret provided when registering LPWan',
```

Comment:
can omit for now, eventually can populte w instrucitons on how to generate TTN client secret for help popover